rest-client/src/main/java/org/sagebionetworks/bridge/rest/AuthenticationHandler.java

```diff
@@ -54,7 +56,11 @@ public AuthenticationHandler(SignIn signIn, UserSessionInfoProvider userSessionI
         }
         tryCount++;
 
-        userSession = userSessionInfoProvider.retrieveSession(signIn);
+        try {
+            userSession = userSessionInfoProvider.retrieveSession(signIn);
+        } catch (ConsentRequiredException e) {
+            LOG.debug("Ignoring 412 during auto-authentication");
```

Comment:
I think you should just let this throw. It would be unexpected and unusual, and would indicate that further calls are going to fail...